django_mysql/models/functions.py

```diff
@@ -260,6 +260,33 @@ def __init__(self, expression, path=None, **extra):
         super(JSONLength, self).__init__(*exprs, output_field=output_field)
 
 
+class JSONSet(Func):
+    function = 'JSON_SET'
+
+    def __init__(self, expression, pairs):
+        from django_mysql.models.fields import JSONField
+
+        if len(pairs) == 0:
+            raise ValueError(
+                'Pairs requires one key/value set defined'
+            )
```

Comment:
`if not len(pairs)` is even shorter :)

Also I'd write `'paris' requires at least one entry`